components/gcp/dataproc/submit_hadoop_job/component.yaml

```diff
@@ -27,12 +27,14 @@ inputs:
   - name: cluster_name
     description: 'Required. The cluster to run the job.'
   - name: main_jar_file_uri
+    default: ''
```

Comment:
Is it really optional?

Reply:
yes, user can put the jar file path in the jarFileUris and specify the mainClass to point to the main entry.

Reply:
yes, user can put the jar file path in the jarFileUris and specify the mainClass to point to the main entry.